target_chains/ethereum/contracts/contracts/executor/ExecutorErrors.sol

```diff
@@ -14,4 +14,6 @@ library ExecutorErrors {
     error DeserializationError();
     // The message is not intended for this contract.
     error InvalidGovernanceTarget();
+    // The message is not intended for a valid contract address.
```

Comment:
the comment here is imprecise. The "message" refers to the wormhole message. I think you want to say "the target address for the contract call is not a contract"